index.html

```diff
@@ -124,5 +124,32 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-section">
+    <div id="join-hiring">
```

Comment:
You could use a class here instead of repeating the styles between two id's later. The other box in the same hiring section has some shared properties, e.g. the border's radius. 

Don't just add id's to elements. Classes are there for reuse and for making things easier. Use them.